schema/schemaspec/schemahcl/hcl.go

```diff
@@ -38,8 +38,20 @@ type (
 	State struct {
 		config *Config
 	}
+	// Evaluator evaluates the Atlas DDL document with the given input and stores it in
+	// the provided object.
+	Evaluator interface {
+		Eval([]byte, interface{}, map[string]string) error
+	}
+	// EvalFunc is the function type that is implemented by the Eval method of the Evaluator interface.
```

Comment:
Can we rephrase this to sth like "allows the use of ordinary functoin as Evaluator", because I am having trouble understanding that sentence :-D